src/components/CopyButton/index.jsx

```diff
@@ -81,10 +82,12 @@ CopyButton.propTypes = {
   textToCopy: PropTypes.string.isRequired,
   onCopyButtonClick: PropTypes.func,
   className: PropTypes.arrayOf(PropTypes.string),
+  'data-identifier': PropTypes.string,
 };
 
 CopyButton.defaultProps = {
   onCopyButtonClick: () => {},
   ariaLabel: 'Copy',
   className: [],
+  'data-identifier': 'copy-button',
```

Comment:
Are we expecting `data-identifier` to be a unique id? It might be better to leave this `undefined` and only add the data attribute to the element when it is defined. Otherwise we might use CopyButton for something else in the future, forget to specify `data-identifier` because it's not required, and then end up with multiple elements with the same `data-identifier="copy-button"`.

Reply:
Ideally, yes, it would be unique. That's a good idea. I'll change it to be undefined.